qutebrowser/config/configdata.yml

```diff
@@ -2241,9 +2241,6 @@ bindings.default:
       Sb: open qute://bookmarks#bookmarks
       Sq: open qute://bookmarks
       Sh: open qute://history
-      xx: config-cycle statusbar.hide ;; config-cycle tabs.show always switching
-      xt: config-cycle tabs.show always switching
-      xb: config-cycle statusbar.hide
```

Comment:
I think that this falls into the same category (of being easy to accidentaly hit for some people). What do you think?

Reply:
Yeah, let's remove them all :wink: